--- tests/tests.lisp	Sun Jan  5 14:09:18 2003
+++ debian/tests.lisp	Thu Mar  6 07:20:17 2003
@@ -23,14 +23,15 @@
 (setq *collect-errors* nil)
 (time 
  (sloop with errs = '() for testv in 
-	'(
-	  "rtest1" "rtest1a" "rtest2" "rtest3" "rtest4" "rtest5"
-	  "rtest6" "rtest6a" "rtest6b" "rtest7"
-	  "rtest8"
-	  "rtest9"
-	  "rtest9a" "rtest10" "rtest11" "rtest12" "rtest13" "rtest13s"
-	  "rtest14" "rtest15" "rexamples" "rtestode" "rtestflatten"
-	  )
+	(mapcar (lambda (x) (concatenate 'string "/usr/share/maxima/@MVERS@/tests/" x))
+		'(
+		  "rtest1" "rtest1a" "rtest2" "rtest3" "rtest4" "rtest5"
+		  "rtest6" "rtest6a" "rtest6b" "rtest7"
+		  "rtest8"
+		  "rtest9"
+		  "rtest9a" "rtest10" "rtest11" "rtest12" "rtest13" "rtest13s"
+		  "rtest14" "rtest15" "rexamples" "rtestode" "rtestflatten"
+		  ))
 	do
 	(format t "~%Testing ~a.mac" testv)
 	(or (errset
