--- xmaxima/xmaxima	Thu Mar  6 07:47:36 2003
+++ xmaxima/xmaxima.p	Thu Mar  6 07:48:28 2003
@@ -14237,7 +14237,7 @@
     $m add command -underline 0 \
 	-state $state \
 	-label {Run Tests} \
-	-command "sendMaxima $text {:lisp (progn (\#+gcl si::chdir \#+clisp ext:cd \#+cmu unix:unix-chdir \"$dir\")(load \"tests.lisp\"))\n}"
+	-command "sendMaxima $text {:lisp (load \"/usr/share/maxima/@MVERS@/tests/tests.lisp\")\n}"
 
 
     # Add a Help menubutton
